.github/etc/dictionary.txt

```diff
@@ -1,2 +1,5 @@
 capricornis
 crispus
+kubeconform
+textlint
+hadolint
```

Comment:
Consider defining the purpose or usage of the added words kubeconform, textlint, and hadolint.